beluga_tools/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>beluga_tools</name>
+  <version>2.0.2</version>
+
+  <description>Set of miscellaneous tools to work with beluga.</description>
+
+  <maintainer email="serraramiro@ekumenlabs.com">Ramiro Serra</maintainer>
+
+  <license>Apache License 2.0</license>
+
+
+  <!-- The released debian for h5py package when on humble seems to have a bug where it's incompatible with numpy's 2.0.0 version.
+  We will install newest package version using pip in the development images for this specific distro.
+  See:
+  https://github.com/h5py/h5py/issues/2353
+  -->
+  <!-- <exec_depend condition="$ROS_DISTRO != 'humble'">python3-h5py</exec_depend> -->
+  <exec_depend condition="$ROS_DISTRO != 'humble'">python3-h5py</exec_depend>
```

Comment:
@serraramiro1 meta: we should also check for `$ROS_VERSION == 2` and not just assume it's all ROS 2.

Reply:
Done